dockstore-client/pom.xml

```diff
@@ -249,6 +254,12 @@
             <version>1.1.1</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.glassfish.jersey.media</groupId>
+            <artifactId>jersey-media-multipart</artifactId>
+            <version>2.25.1</version>
```

Comment:
There's a property for this to keep jersey dependencies in sync 
https://github.com/ga4gh/dockstore/blob/develop/pom.xml#L38

Reply:
Done now using property